specification/machinelearningservices/data-plane/Microsoft.MachineLearningServices/preview/2019-05-01/modelManagement.json

```diff
@@ -0,0 +1,4613 @@
+        },
+        "autoDelete": {
+          "description": "Whether the image will be automatically deleted with the last service using it.",
+          "type": "boolean"
+        },
+        "imageType": {
+          "description": "The type of the image.",
+          "enum": [
+            "Docker"
+          ],
+          "type": "string",
+          "example": "Docker",
+          "x-ms-enum": {
+            "name": "ImageType",
+            "modelAsString": true
+          }
+        },
+        "imageFlavor": {
+          "description": "The flavor of the image.",
+          "enum": [
+            "WebApiContainer",
+            "BatchContainer",
+            "IoTContainer",
+            "BrainwavePackage",
+            "AccelContainer",
+            "BrainwaveContainer",
+            "UserProvidedContainer"
+          ],
+          "type": "string",
+          "example": "WebApiContainer",
+          "x-ms-enum": {
+            "name": "ImageFlavor",
+            "modelAsString": true
+          }
+        },
+        "creationState": {
+          "description": "The state of the operation.",
+          "enum": [
+            "NotStarted",
+            "Running",
+            "Cancelled",
+            "Succeeded",
+            "Failed",
+            "TimedOut"
+          ],
+          "type": "string",
+          "example": "Running",
+          "x-ms-enum": {
+            "name": "AsyncOperationState",
+            "modelAsString": true
+          }
+        },
+        "error": {
+          "$ref": "#/definitions/ModelErrorResponse",
+          "description": "The error response."
+        },
+        "modelIds": {
+          "description": "The list of model Ids.",
+          "uniqueItems": false,
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "modelDetails": {
+          "description": "The list of models.",
+          "uniqueItems": false,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Model"
+          }
+        },
+        "imageLocation": {
+          "description": "The Image location string.",
+          "type": "string"
+        },
+        "imageBuildLogUri": {
+          "description": "The Uri to the image build logs.",
+          "type": "string"
+        },
+        "operationId": {
+          "description": "The ID of the asynchronous operation for this image.",
+          "type": "string"
+        }
+      }
+    },
+    "ModelErrorResponse": {
+      "description": "The Model Management Service Error object.",
+      "type": "object",
+      "properties": {
+        "code": {
+          "description": "The error code.",
+          "type": "string"
+        },
+        "statusCode": {
+          "format": "int32",
+          "description": "The HTTP status code.",
+          "type": "integer"
+        },
+        "message": {
+          "description": "The error message.",
+          "type
```

Comment:
Why are we using an allOf with an inline definition of the schema? I'd expect the second element to be in the `properties`  of the object definition...

Reply:
Need more clarification.

Reply:
Should be moved to Properties below but ok for now.

Reply:
No further action for now.